src/components/inputs/formik-text-editor.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { TextEditor } from "openstack-uicore-foundation/lib/components";
+import { useFormikContext } from "formik";
+
+const FormikTextEditor = ({ name, ...props }) => {
+  const { values, errors, touched, setFieldValue, setFieldTouched } =
+    useFormikContext();
+
+  return (
+    <TextEditor
```

Comment:
this shhould be the new text editor check that we already changed all the text editors at show admin